Add 6.2.5 release notes (#2983)
sensu/sensu-docs: content/sensu-go/6.2/release-notes.md

---

## 6.2.5 release notes

**February 2, 2021** &mdash; The latest release of Sensu Go, version 6.2.5, is now available for download.

> This patch fixes a bug regarding event occurrence watermark property. This bug interfered with expected behavior when using event filters like the popular fatigue check filter.

Reviewer: ```suggestion
This patch fixes a bug regarding the event occurrences_watermark property.
This bug interfered with the property's expected behavior when using event filters like the popular fatigue check filter.
```

See the [upgrade guide][1] to upgrade Sensu to version 6.2.5.

**FIXES:**

> - ([Commercial feature][193]) Fixed a bug where occurrences_watermark wouldn't increment for non-zero events when using the PostgreSQL datastore.

Reviewer: ```suggestion
- ([Commercial feature][193]) Fixed a bug that prevented occurrences_watermark from incrementing for non-zero events when using the PostgreSQL datastore.
```
